### PR TITLE
ENH: pcds-3.2.0

### LINKED ISSUE #102: Add LUCID
https://github.com/pcdshub/lucid

### LINKED ISSUE #107: pmgr and lucid are incompatible
pmgr is based on a modified build of pyqt due to a feature regression from pyqt4 to pyqt5. The version number is too low for a dependency of lucid.

### PR DESCRIPTION
**Pull request by ZLLentz**
First try?

The goal of this release is to add `lucid` and to switch to a `conda-forge` priority over defaults.

closes #102 
closes #107

### CODE REVIEW

File: envs/pcds/env.yaml [hunk 1]
```diff
@@ -28,22 +27,22 @@ dependencies:
...
+  - click=7.1.1
+  - cloudpickle=1.3.0
   - clyent=1.2.2
   - colorcet=2.0.2
   - coloredlogs=10.0
   - cookiecutter=1.7.0
   - coverage=5.0
   - cryptography=2.8
-  - curio=0.9
+  - curio=1.1
```
Comment (klauer) [inline envs/pcds/env.yaml]: caproto pins curio to <1.0 due to API changes: https://github.com/caproto/caproto/blob/0a8b57ee65416ffb128f001d8047daa192eeab6e/test-requirements.txt#L3

Comment (ZLLentz) [reply envs/pcds/env.yaml]: Our env is prioritizing curio over caproto, causing this to get updated instead. Should I pin to the newest available caproto and rebuild?

Comment (klauer) [reply envs/pcds/env.yaml]: Older caproto won't work with newer curio - its deps just are more lax. Should pin to newest caproto, I think.

Comment (ZLLentz) [reply envs/pcds/env.yaml]: will do

File: envs/pcds/env.yaml [hunk 2]
```diff
@@ -195,33 +196,33 @@ dependencies:
...
   - ptyprocess=0.6.0
   - py=1.8.1
-  - py-opencv=3.4.2
   - pyaudio=0.2.11
   - pyca=3.1.1
   - pycodestyle=2.5.0
-  - pycparser=2.19
+  - pycparser=2.20
   - pyct=0.4.6
   - pydm=1.8.0
```
Comment (hhslepicka) [inline envs/pcds/env.yaml]: I recommend bumping to 1.9.0 as many fixes were added.

File: envs/pcds/env.yaml [hunk 3]
```diff
@@ -232,55 +233,61 @@ dependencies:
...
+  - tk=8.6.10
+  - toml=0.10.0
   - toolz=0.10.0
   - tornado=4.5.3
-  - tqdm=4.42.0
+  - tqdm=4.43.0
   - traitlets=4.3.3
   - transfocate=0.3.2
+  - typed-ast=1.4.1
   - typhos=0.6.0
```
Comment (hhslepicka) [inline envs/pcds/env.yaml]: typhos=0.7.0

**Comment by ZLLentz** [comment]
Next time I'm going to create a latest-versions script that pins all of our releases automatically...

File: envs/pcds/env.yaml [hunk 4]
```diff
@@ -61,129 +60,131 @@ dependencies:
...
-  - graphviz=2.40.1
-  - gst-plugins-base=1.14.0
-  - gstreamer=1.14.0
+  - graphviz=2.42.3
+  - gst-plugins-base=1.14.5
+  - gstreamer=1.14.5
   - h5py=2.8.0
-  - happi=1.2.1
-  - harfbuzz=1.8.8
+  - happi=1.3.0
```
Comment (hhslepicka) [inline envs/pcds/env.yaml]: happi=1.4.0

**Comment by ZLLentz** [comment]
I'm going to need to look at `lightpath`, the tests fail due to `happi` changes but they are likely to be cosmetic

**Comment by ZLLentz** [comment]
The limiting failure is on running `pydm --help` of all things:
`bash: /u1/zlentz/miniconda/envs/pcds-3.2.0/bin/pydm: /home/conda/feedstock_root/build_artifacts/pydm_1584229929852/_h_env_placehold: bad interpreter: No such file or directory`

**Comment by ZLLentz** [comment]
@hhslepicka I think the pydm conda forge build has an issue

**Comment by hhslepicka** [comment]
How did I close this?

**Comment by ZLLentz** [comment]
It probably needed to be kicked/restarted for the travis anyway!

**Comment by hhslepicka** [comment]
🤞

**Pull request merged**